docs/config.md

```diff
@@ -0,0 +1,266 @@
+# Configure Redoc
+
+Getting your documentation just right is important, and Redoc comes with many configuration options to help you succeed in that mission.
+
+Each deployment type has documentation on how to options in its context. This page lists all the options you can use with Redoc.
```

Comment:
This sentence is awkward. Do you mean how-to options? In the context of the deployment type?

Reply:
I think it's missing a word :laughing: Let me try again ...

Reply:
Is that better?